backend/src/test/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhirTest.java

```diff
@@ -229,14 +228,8 @@ private InputStream getJsonStream(String jsonFile) {
     return BulkUploadResultsToFhirTest.class.getClassLoader().getResourceAsStream(jsonFile);
   }
 
-  private String readJsonStream(InputStream is) throws IOException {
-    BufferedReader reader = new BufferedReader(new InputStreamReader(is));
-    StringBuilder stringBuilder = new StringBuilder();
-    String line;
-    while ((line = reader.readLine()) != null) {
-      stringBuilder.append(line);
-    }
-    return stringBuilder.toString();
```

Comment:
This method was eating all new lines in the input stream.

Reply:
Good catch for the multi row test files!